app/app.go

```diff
@@ -200,6 +200,18 @@ var (
 // Verify app interface at compile time
 var _ simapp.App = (*IritaApp)(nil)
 
+var AddModule AddModuleFun
+
+var AnteHandler AnteHandlerFun
+
+var UpgradePlan RegisterUpgradePlanFun
```

Comment:
Can these global variables be turned into Option parameters?

Reply:
ok